oemof_b3/tools/data_processing.py

```diff
@@ -0,0 +1,121 @@
+import pandas as pd
+import numpy as np
+
+
+def check_consistency_timeindex(df, index, name):
+    """
+    This function assert that values of a column in a stacked DataFrame are same
+    for all time steps
+
+    Parameters
+    ----------
+    index : string
+        Index of values to be checked in the DataFrame
+    name : string
```

Comment:
'name' is only used for the error message, no other function. Could we drop it?

Reply:
Done with commit https://github.com/rl-institut/oemof-B3/pull/47/commits/8b0ea2db7abcb99128185834960a17a2a75efb79.